mutiny-core/src/nodemanager.rs

```diff
@@ -1238,6 +1241,9 @@ impl<S: MutinyStorage> NodeManager<S> {
             return Err(e);
         }
 
+        // set has synced to true
+        self.has_done_initial_ldk_sync.swap(true, Ordering::SeqCst);
+
```

Comment:
Payments before syncing LDK isn't the problem, the problem is BDK chain tip is not up to date. This needs to be moved down.

Reply:
I'm confused, how would bdk effect ldk? They don't talk to each other

Reply:
how is the chain tip retrieved?

Reply:
ldk syncs it